__tests__/helpers/Solo.ts

```diff
@@ -5,6 +5,7 @@ import { address, ConfirmationType, SoloOptions } from '../../src/types';
 const soloOptions: SoloOptions = {
   confirmationType: ConfirmationType.Confirmed,
   testing: true,
+  defaultGas: '4000000',
```

Comment:
necessary due to the new ganache-cli not returning error messages properly when estimating gas. so don't estimate gas in the tests anymore